setup.py

```diff
@@ -48,11 +72,15 @@ def find_version(*file_paths):
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3 :: Only',
     ],
-
     # Package info
-    packages=find_packages(exclude=('test', 'test.*')),
-
-    zip_safe=True,
+    packages=find_packages(exclude=('test*', 'build_tools*')),
+    zip_safe=False,
+    # Extension info
+    ext_modules=setup_helpers.get_ext_modules(),
+    cmdclass={
+        'build_ext': setup_helpers.BuildExtension.with_options(no_python_abi_suffix=True),
```

Comment:
Why is this abi option necessary?

Also, I'd add a comment that references the issue around dropping symbols we recently referenced (I can dig it up if this doesn't ring a bell).

Reply:
ah not necessary. It's a reminding custom from what I followed in `torchvision`. (`_C.so` notation.)
I will remove it.

Reply:
Actually, it turned out that we need to provide the full path to `so` file, so removing abi suffix makes it easier to locate the library file.